GameData/KerbalismConfig/System/Science/Configure.cfg

```diff
@@ -472,6 +472,84 @@
 // Crew experiment Selection
 // ============================================================================
 
+// Cockpits
+@PART[*]:HAS[#capsuleTier[Cockpit]]:NEEDS[FeatureScience,RP-0]:AFTER[RP-0-Kerbalism]
+{
+	@MODULE[HardDrive]
+	{
+		@sampleCapacity += 1
+	}
+
+	MODULE
+	{
+		name = Configure
+		title = Crew Science
+		slots = 1
+		
+		SETUP
+		{
+			name = None
+			desc = Empty slot for mass and cost savings, should you not require any experiments installed.
+		}
+
+		SETUP
+		{
+			name = Supersonic flight
+			desc = //fixme
+			mass = 0.004
+			
+			MODULE
+			{
+			  type = Experiment
+			  id_field = experiment_id
+			  id_value = RP0-SupersonicLow1
+			}
+		}
+		SETUP
+		{
+			name = Mach 2 flight
+			desc = //fixme
+			mass = 0.004
+			tech = supersonicDev
+			
+			MODULE
+			{
+			  type = Experiment
+			  id_field = experiment_id
+			  id_value = RP0-SupersonicLow2
+			}
+		}
+		SETUP
+		{
+			name = High altitude flight
+			desc = //fixme
+			mass = 0.004
+			tech = supersonicFlightRP0
+			
+			MODULE
+			{
+			  type = Experiment
+			  id_field = experiment_id
+			  id_value = RP0-SupersonicHigh1
+			}
+		}
+		SETUP
+		{
+			name = Hypersonic flight
+			desc = //fixme
+			mass = 0.004
+			tech = supersonicFlightRP0
```

Comment:
Wouldn't a Hypersonic Flight experiment make more sense to unlock in the Hypersonic Flight r&d node?

Reply:
High altitude is already there, no? If it's not, it was a mistake on my part. About Hypersonic Flight node, I could do that yeah... my idea for that is X-15-like flights.

Reply:
If we do move it to hypersonic flight then 20 sci isn't worth much at all by then.

Reply:
high altitude: sorry yes, my turn to confuse the first two supersonic nodes